libraries/AP_AIS/AP_AIS.cpp

```diff
@@ -0,0 +1,782 @@
+/*
```

Comment:
Just a small thing which is that this commit should be prefixed with "AP_AIS:"

Reply:
fixed